doc/sphinx/changes.rst

```diff
@@ -853,6 +853,64 @@ Changes in 8.12+beta1
   (`#9803 <https://github.com/coq/coq/pull/9803>`_,
   by Laurent Théry and Michael Soegtrop).
 
+**Reference manual**
+
+- **Changed:**
+  The reference manual has been restructured.  In the new version,
+  there are less top-level chapters, and, in the HTML format, chapters
+  are split on multiple pages.  This is still a work-in-progress and
+  further restructuring is expected in the next versions of Coq
+  (`CEP#43 <https://github.com/coq/ceps/pull/43>`_, implemented in
+  `#11601 <https://github.com/coq/coq/pull/11601>`_,
+  `#11871 <https://github.com/coq/coq/pull/11871>`_,
+  `#11914 <https://github.com/coq/coq/pull/11914>`_,
+  `#12148 <https://github.com/coq/coq/pull/12148>`_,
+  `#12172 <https://github.com/coq/coq/pull/12172>`_,
+  `#12239 <https://github.com/coq/coq/pull/12239>`_
+  and `#12330 <https://github.com/coq/coq/pull/12330>`_,
+  effort inspired by Matthieu Sozeau, led by Théo Zimmermann, with
+  help and reviews of Jim Fehrle, Clément Pit-Claudel and others).
+- **Changed:**
+  Most of the grammar is now presented using the same box mechanism as
+  was already used to present commands and tactics since Coq 8.8 and
+  which is documented in :ref:`syntax-conventions`
+  (`#11183 <https://github.com/coq/coq/pull/11183>`_,
+  `#11314 <https://github.com/coq/coq/pull/11314>`_,
+  `#11423 <https://github.com/coq/coq/pull/11423>`_,
+  `#11705 <https://github.com/coq/coq/pull/11705>`_,
+  `#11718 <https://github.com/coq/coq/pull/11718>`_,
+  `#11720 <https://github.com/coq/coq/pull/11720>`_
+  and `#11961 <https://github.com/coq/coq/pull/11961>`_, by Jim
+  Fehrle, with reviews of Théo Zimmermann).
+- **Added:**
+  A glossary of terms and an index of attributes
+  (`#11869 <https://github.com/coq/coq/pull/11869>`_,
+  `#12150 <https://github.com/coq/coq/pull/12150>`_
+  and `#12224 <https://github.com/coq/coq/pull/12224>`_,
+  by Jim Fehrle and Théo Zimmermann,
+  with reviews of Clément Pit-Claudel)
+- **Added:**
+  A selector that allows to switch between versions of the reference
+  manual (`#12286 <https://github.com/coq/coq/pull/12286>`_, by
+  Clément Pit-Claudel).
+- **Fixed:**
+  A large part of the documented syntax has been fixed to correspond
+  more accurately to what is actually implemented, thanks to an
+  internal `doc_grammar` tool introduced for this purpose
```

Comment:
```suggestion
  Most of the documented syntax has been thoroughly updated to make it accurate and
  easily understood.  This was done using a semi-automated `doc_grammar`
  tool introduced for this purpose and through significant revisions to the text.
```

Still very understated--getting an accurate, readable grammar is a big step forward.
The pre-change grammar was really bad and incomplete.
Still a good bit of syntax that's not explained (e.g. in the terms chapter).

Will we mention the doc improvements (and the plans for subsequent improvements) more prominently, such as in the email announcing the release?

Reply:
Sorry, I missed most of your comments last time:

>Will we mention the doc improvements (and the plans for subsequent improvements) more prominently, such as in the email announcing the release?

There's a summary to be written above, which highlights a selection of changes. The documentation changes in 8.12 are clearly one of the highlights to be done.

The e-mail announcing the release will repeat these highlights. But in addition, the 8.12+beta1 announcement should include a call for beta readers of the manual and for help in improving the presentation further. I'll get back to you for help writing this call.